airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2502,6 +2551,53 @@ components:
         - auto
         - us
         - eu
+    # BuilderProject
+    ConnectorBuilderProjectId:
+      type: string
+      format: uuid
+    ConnectorBuilderProjectDetails:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+        draftManifest:
+          type: object
+          additionalProperties: true
+    ConnectorBuilderProjectIdWithWorkspaceId:
+      type: object
+      required:
+        - workspaceId
+        - builderProjectId
+      properties:
+        workspaceId:
+          $ref: "#/components/schemas/WorkspaceId"
```

Comment:
Using this, it means that we are binding with the definition of a "workspaceId" in the platform. This means that if platform wants to change that, we will be impacted. Is this something that makes sense to us?

Reply:
The `WorkspaceId` schema is just a uuid-string at the moment to make sure this compiles to a `UUID` member, but in general I think it makes sense to get an error if this assumption changes - after all we also have a `workspaceId` column in the builder project model.